config/src/config/safety_rules_config.rs

```diff
@@ -40,6 +41,7 @@ impl Default for SafetyRulesConfig {
             // Default value of 30 seconds for a timeout
             network_timeout_ms: 30_000,
             enable_cached_safety_data: true,
+            initial_safety_rules_config: InitialSafetyRulesConfig::None,
```

Comment:
nit: you could make this optional so that you don't need an explicit "None" value?

Reply:
I followed the current pattern in the event we need to have different inputs.